src/sentry/notifications/helpers.py

```diff
@@ -91,7 +91,7 @@ def _get_setting_mapping_from_mapping(
 
     return {
         provider: _get_notification_setting_default(provider, type, user=user)
-        for provider in [ExternalProviders.EMAIL]
+        for provider in [ExternalProviders.EMAIL, ExternalProviders.SLACK]
```

Comment:
I wasn't sure if I should do this or just get all `ExternalProviders` enum values

Reply:
You can remove the `XXX(CEO)` comment now.

Reply:
the comment was about if we change the front end to allow a user to set Slack notifications for project A, and email notifications for project B, which we don't allow, so I think it still stands.

Reply:
Yeah we should just do all of them here.
```python
from sentry.notifications.notify import notification_providers
...
for provider in notification_providers()
```

Reply:
Maybe a separate Notification Platform enum? If this list is replicated in multiple places it'd make for easier changes down the line.

Reply:
^^ guess im late